INFO:     Will watch for changes in these directories: ['C:\\Users\\blaze\\Desktop\\revize_program\\Python\\revize_web\\revize-backend']
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [21164] using StatReload
Process SpawnProcess-1:
Traceback (most recent call last):
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 22, in import_from_string
    raise exc from None
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\main.py", line 3, in <module>
    from routers.auth import router as auth_router, get_current_user
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\routers\auth.py", line 11, in <module>
    from utils.security import hash_password, verify_password, create_access_token
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\utils\security.py", line 15, in <module>
    import bcrypt
ModuleNotFoundError: No module named 'bcrypt'
WARNING:  StatReload detected changes in 'routers\admin.py'. Reloading...
Process SpawnProcess-2:
Traceback (most recent call last):
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 22, in import_from_string
    raise exc from None
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\main.py", line 3, in <module>
    from routers.auth import router as auth_router, get_current_user
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\routers\auth.py", line 11, in <module>
    from utils.security import hash_password, verify_password, create_access_token
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\utils\security.py", line 15, in <module>
    import bcrypt
ModuleNotFoundError: No module named 'bcrypt'
WARNING:  StatReload detected changes in 'routers\admin.py'. Reloading...
Process SpawnProcess-3:
Traceback (most recent call last):
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 22, in import_from_string
    raise exc from None
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\main.py", line 3, in <module>
    from routers.auth import router as auth_router, get_current_user
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\routers\auth.py", line 11, in <module>
    from utils.security import hash_password, verify_password, create_access_token
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\utils\security.py", line 15, in <module>
    import bcrypt
ModuleNotFoundError: No module named 'bcrypt'
WARNING:  StatReload detected changes in 'routers\admin.py'. Reloading...
Process SpawnProcess-4:
Traceback (most recent call last):
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 22, in import_from_string
    raise exc from None
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\main.py", line 3, in <module>
    from routers.auth import router as auth_router, get_current_user
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\routers\auth.py", line 11, in <module>
    from utils.security import hash_password, verify_password, create_access_token
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\utils\security.py", line 15, in <module>
    import bcrypt
ModuleNotFoundError: No module named 'bcrypt'
WARNING:  StatReload detected changes in 'routers\admin.py'. Reloading...
Process SpawnProcess-5:
Traceback (most recent call last):
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 22, in import_from_string
    raise exc from None
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\main.py", line 3, in <module>
    from routers.auth import router as auth_router, get_current_user
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\routers\auth.py", line 11, in <module>
    from utils.security import hash_password, verify_password, create_access_token
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\utils\security.py", line 15, in <module>
    import bcrypt
ModuleNotFoundError: No module named 'bcrypt'
WARNING:  StatReload detected changes in 'routers\revisions.py'. Reloading...
Process SpawnProcess-6:
Traceback (most recent call last):
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 22, in import_from_string
    raise exc from None
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\main.py", line 3, in <module>
    from routers.auth import router as auth_router, get_current_user
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\routers\auth.py", line 11, in <module>
    from utils.security import hash_password, verify_password, create_access_token
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\utils\security.py", line 15, in <module>
    import bcrypt
ModuleNotFoundError: No module named 'bcrypt'
WARNING:  StatReload detected changes in 'routers\admin.py'. Reloading...
Process SpawnProcess-7:
Traceback (most recent call last):
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 22, in import_from_string
    raise exc from None
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\main.py", line 3, in <module>
    from routers.auth import router as auth_router, get_current_user
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\routers\auth.py", line 11, in <module>
    from utils.security import hash_password, verify_password, create_access_token
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\utils\security.py", line 15, in <module>
    import bcrypt
ModuleNotFoundError: No module named 'bcrypt'
WARNING:  StatReload detected changes in 'routers\admin.py'. Reloading...
Process SpawnProcess-8:
Traceback (most recent call last):
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 22, in import_from_string
    raise exc from None
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\main.py", line 3, in <module>
    from routers.auth import router as auth_router, get_current_user
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\routers\auth.py", line 11, in <module>
    from utils.security import hash_password, verify_password, create_access_token
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\utils\security.py", line 15, in <module>
    import bcrypt
ModuleNotFoundError: No module named 'bcrypt'
WARNING:  StatReload detected changes in 'routers\admin.py'. Reloading...
Process SpawnProcess-9:
Traceback (most recent call last):
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 22, in import_from_string
    raise exc from None
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\main.py", line 3, in <module>
    from routers.auth import router as auth_router, get_current_user
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\routers\auth.py", line 11, in <module>
    from utils.security import hash_password, verify_password, create_access_token
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\utils\security.py", line 15, in <module>
    import bcrypt
ModuleNotFoundError: No module named 'bcrypt'
WARNING:  StatReload detected changes in 'routers\admin.py'. Reloading...
Process SpawnProcess-10:
Traceback (most recent call last):
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 22, in import_from_string
    raise exc from None
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\main.py", line 3, in <module>
    from routers.auth import router as auth_router, get_current_user
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\routers\auth.py", line 11, in <module>
    from utils.security import hash_password, verify_password, create_access_token
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\utils\security.py", line 15, in <module>
    import bcrypt
ModuleNotFoundError: No module named 'bcrypt'
WARNING:  StatReload detected changes in 'routers\admin.py'. Reloading...
Process SpawnProcess-11:
Traceback (most recent call last):
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 22, in import_from_string
    raise exc from None
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\main.py", line 3, in <module>
    from routers.auth import router as auth_router, get_current_user
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\routers\auth.py", line 11, in <module>
    from utils.security import hash_password, verify_password, create_access_token
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\utils\security.py", line 15, in <module>
    import bcrypt
ModuleNotFoundError: No module named 'bcrypt'
WARNING:  StatReload detected changes in 'routers\admin.py'. Reloading...
Process SpawnProcess-12:
Traceback (most recent call last):
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 22, in import_from_string
    raise exc from None
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\main.py", line 3, in <module>
    from routers.auth import router as auth_router, get_current_user
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\routers\auth.py", line 11, in <module>
    from utils.security import hash_password, verify_password, create_access_token
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\utils\security.py", line 15, in <module>
    import bcrypt
ModuleNotFoundError: No module named 'bcrypt'
WARNING:  StatReload detected changes in 'models.py'. Reloading...
Process SpawnProcess-13:
Traceback (most recent call last):
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 71, in serve
    await self._serve(sockets)
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\server.py", line 78, in _serve
    config.load()
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 22, in import_from_string
    raise exc from None
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\venv\Lib\site-packages\uvicorn\importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\blaze\anaconda3\envs\revize-web\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\main.py", line 3, in <module>
    from routers.auth import router as auth_router, get_current_user
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\routers\auth.py", line 11, in <module>
    from utils.security import hash_password, verify_password, create_access_token
  File "C:\Users\blaze\Desktop\revize_program\Python\revize_web\revize-backend\utils\security.py", line 15, in <module>
    import bcrypt
ModuleNotFoundError: No module named 'bcrypt'
